pose_format/utils/holistic.py

```diff
@@ -37,13 +38,14 @@ def component_points(component, width: int, height: int, num: int):
 def body_points(component, width: int, height: int, num: int):
     if component is not None:
         lm = component.landmark
+        # TODO: * width?
```

Comment:
From https://github.com/J22Melody/pose-pipelines/blob/holistic/example.mediapipe.json (from Surrey), they save the original numbers without multiplying width and/or height. 

Do you think it would affect the downstream tasks?